jsonp/1.1/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta JSON Processing Specification 1.1"
+date: 2019-07-19T11:00:00+0100
```

Comment:
The time component is not needed, and probably not useful in this context.

Reply:
We should probably update the template so that we can remove this time component comment...

Reply:
Already updated: https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md

Reply:
Done